app/assets/javascripts/discourse/widgets/hamburger-categories.js.es6

```diff
@@ -5,7 +5,14 @@ import { number } from 'discourse/lib/formatter';
 createWidget('hamburger-category', {
   tagName: 'li.category-link',
 
+  buildAttributes(attrs) {
```

Comment:
I was trying to keep how to target a class consistent, but I can look at switching it. It'll involve a bit more code as I'll have to build the slug out looking at parent/child.